Sources/CorePayments/Networking/HTTPResponse.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+struct HTTPResponse {
+    
+    let status: Int
+    let body: Data?
+    
+    var isSuccessful: Bool { (200...300).contains(status) }
```

Comment:
```suggestion
    var isSuccessful: Bool { (200..<300).contains(status) }
```
Slight 🧶 we probably won't get many 300s but this will make sure to keep [HTTP 300](https://httpstatusdogs.com/300-multiple-choices) out of the successful response range.

Reply:
Good catch! Updated in this commit (634b59a12de8878414ba6c892723b9fb0350c71d) as well as added unit tests for the `isSuccessful` bool.